### PR TITLE
Fix popup text on map in dark mode

### LINKED ISSUE #1886: "Here" not coming on map in dark theme
**Actual Behaviour**

screenshot from 2018-12-13 02-03-23

**Expected Behaviour**
screenshot from 2018-12-13 02-03-43

**Steps to reproduce it**
open any location on map.

**LogCat for the issue**

**Screenshots of the issue**

**Would you like to work on the issue?**
yes

### PR DESCRIPTION
**Pull request by ananya-agrawal**
Fixes #1886 

Changes: Fixed text color in popup of map

Demo Link: https://pr-1891-fossasia-susi-web-chat.surge.sh
http://subsequent-plantation.surge.sh/

### CODE REVIEW

**Comment by shreyanshdwivedi** [comment]
@ananya-agrawal please refrain yourself from commiting via master branch. Always open a new branch for a new PR :)

**Comment by garvitkhandelwal** [comment]
@ananya-agrawal  Why are there two demo links?

**Comment by AakashMallik** [comment]
Update branch.

**Comment by ananya-agrawal** [comment]
@AakashMallik done

**Comment by AakashMallik** [comment]
@ananya-agrawal I was unable to test the charges you made because of this
screenshot-pr-1891-fossasia-susi-web-chat surge sh-2018 12 23-19-11-52
Could you take some time out and see why this is happening.

**Comment by ananya-agrawal** [comment]
@Akshat-Jain @AakashMallik  please review the pr

File: src/components/ChatApp/ChatApp.css [hunk 1]
```diff
@@ -619,6 +619,10 @@ textarea {
   margin: 0 auto;
 }
 
+.leaflet-popup-content {
+  color: black;
```
Comment (AakashMallik) [inline src/components/ChatApp/ChatApp.css]: Use hex value

**Pull request merged**